Examples/ComplexVaporExample/Package.swift

```diff
@@ -0,0 +1,28 @@
+// swift-tools-version:5.2
+import PackageDescription
+
+let package = Package(
+    name: "ComplexVaporExample",
+    platforms: [
+        .macOS(.v10_15)
+    ],
+    dependencies: [
+        .package(url: "https://github.com/vapor/vapor", .upToNextMajor(from: "4.7.0")),
+        .package(url: "https://github.com/vapor/leaf", .exact("4.0.0-rc.1.2")),
```

Comment:
they haven't tagged a final 4.0 of leaf yet, so pinning to exact RC version just in case things change